intelligence/dev/build-example-docs.py

```diff
@@ -0,0 +1,150 @@
+# Copyright 2025 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Build the examples docs."""
+
+import os
+import re
+import shutil
+
+from pathlib import Path
+
+ROOT = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
+INDEX = os.path.join(ROOT, "docs", "source", "examples.rst")
+
+initial_text = """
+Examples
+========
+
+Below you will find a list of Flower Intelligence examples for Node.js and for
+web apps  running in the browser (as a tab or WebExtension).
+
+Node.js Examples
+----------------
+
+Those examples will run in the terminal and are mostly there to showcase some
+features with very low overhead. You'll find more instruction in the
+respective pages.
```

Comment:
Should there be a page for Web Examples?

Reply:
It'd be great to have at least two - maybe the minimal chat and the upcoming Next.js chat?